src/pipx/colors.py

```diff
@@ -1,13 +1,14 @@
 import sys
 from typing import Callable
 
-from pipx.constants import WINDOWS
-
 PRINT_COLOR = sys.stdout.isatty()
```

Comment:
Just a style nit: Could you place the `PRINT_COLOR` line after the `try`...`except` stanza (I like to put all the imports first as much as is possible.)  and I think that should do it.